.github/workflows/rust.yml

```diff
@@ -60,7 +60,7 @@ jobs:
     strategy:
       matrix:
         rust:
-          - 1.29.0
```

Comment:
If we want CI to pass with all commits, this one should go to the very front of the PR.

Reply:
Done, Thanks for noticing :)